README.md

```diff
@@ -2,72 +2,67 @@
 
 # Project Name
 
-> Description the project.
+> Awesome books - Javascript Single Page Application
```

Comment:
- [ ] Kindly make sure to add a description of the project in about 3- 4 sentences like "`In this project, we have created a fully functional website for Awesome books where we have added three sections: Books list, Add book form, a new Contact info section with a Navigation bar, and a Footer.`"

Reply:
resolved